src/LSODA.cpp

```diff
@@ -379,9 +383,9 @@ void LSODA::lsoda(LSODA_ODE_SYSTEM_TYPE f, const size_t neq, vector<double> &y,
       atoli = atol_[1];
       for (size_t i = 1; i <= n; ++i)
       {
-        if (itol_ >= 3)
+        if (itol >= 3)
           rtoli = rtol_[i];
-        if (itol_ == 2 || itol_ == 4)
+        if (itol == 2 || itol == 4)
           atoli = atol_[i];
```

Comment:
This is all code from the ode solver; we _could_ eventually support `itol=2` and `itol=3` with some different code to check the inputs.